EdgeOfEmpireBot/EdgeOfEmpireBot/MessageHandlers/GeneralMessageHandler/GeneralMessageHandler.cs

```diff
@@ -7,6 +7,8 @@ namespace HK47.MessageHandlers;
 
 public class GeneralMessageHandler(IMessageService messageService, IDataService dataService) : IGeneralMessageHandler
 {
+    private readonly string[] ListOfRpgs = ["edge", "masks", "sprawl"]; // This contains example rpgs
```

Comment:
so is the plan to expand this from an arrary, to potential enum or something?

Reply:
This can be an Array or it can be an enum. This is simply first iteration.
I opted for an array as you can just run a array.contains(value) on it.

Reply:
Happy to leave for now, do you think we need an issue for improvments or probs fine?